### DISCUSSION TITLE
Alist V2 支持的"忽略父级密码"的功能在V3中不支持了吗?

### DISCUSSION THREAD

**Discussion by jsjcjsjc** [AlistGo/alist#3199] [Q&A] [answered]
你好
感谢你的应用,最近测试了一下V3,发现Alist V2 支持的"忽略父级密码"的功能在V3中不支持了吗?
似乎现在结尾必须带上类似这样的字符?`?sign=1UQKtSsb35VNFVmDlixK-rornieJS5PUYTGOBD-uGtE=:0`
本来是想做一个简单的图传使用的.
感谢

**Comment by anwen-anyi** [comment]
签名全部 自行关闭

**Comment by WenLiCG** [comment, marked as answer]
在,设置->全局->关闭"签名所有"就可以了
不过目前是默认父级密码
